packages/terra-action-footer/tests/jest/CenteredActionFooter.test.jsx

```diff
@@ -73,5 +96,13 @@ describe('CenteredActionFooter', () => {
         expect(centeredActionFooter.find('ActionFooterContainer').length).toBe(1)
       ));
     });
+
+    describe('with a block center action', () => {
```

Comment:
Do we want to repeat the test name?

Reply:
No each test step should be unique. Good catch.